functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -159,4 +170,5 @@ def add_tasks(conn, task_req_body, user_id):
         task_id = cursor.fetchval()
         logging.info(task_id)
         logging.debug("task with id {} added!".format(task_id))
+        
         return func.HttpResponse(json.dumps({task_id}, default=default), status_code=200, mimetype="application/json")

```

Comment:
currently, it's not returning taskId in a correct format. should be =>
return func.HttpResponse(json.dumps({"taskId": task_id}), status_code=200, mimetype="application/json")